client/src/store/modules/i18n/uk.js

```diff
@@ -27,6 +27,8 @@ export default {
     CALENDAR_INVITATION_HAS_BEEN_SENT:
         'Запрошення до календаря надіслано на вашу електронну адресу',
     CANCEL: 'Скасувати',
+    CANCEL_EVENT_TEXT: 'Are you sure that you want cancel this event?',
+    CANCELLATION_EVENT: 'Cancellation event!',
```

Comment:
Why you don't use Ukraine language?

Reply:
I don't have Ukrainian keyboard layout in OS X